src/Microsoft.PowerShell.Commands.Management/commands/management/Process.cs

```diff
@@ -10,6 +10,7 @@
 using System.Diagnostics; // Process class
 using System.Diagnostics.CodeAnalysis;
 using System.IO;
+using System.Linq;
```

Comment:
Please don't use Linq as our code guideline says. Suggestion is:
```
Array.Empty<CompletionResult>()
``

Reply:
Fixed 5a471f779